### PR TITLE
Feature/cdf/init from jsons

### PR DESCRIPTION
**Pull request by schneiderjan**
Thank you for your contribution to the **Tomni** package.
Before submitting/accepting this Pull Request, please make sure:

## Contributor of code

- [x] Version number is updated
- [x] `HISTORY.rst` is updated
- [x] Function is tested
- [x] `(docstring)` is added or updated
- [ ] All needed package are in [requirements.txt](requirements.txt)

**For new function**

- [x] Functions are named based on [Python naming conventions](https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html)
- [ ] Function is added to apidocx.rst
- [ ] New feature is added to `README.md` subsection `Features`
- [x] New function is imported in all `__init__.py` that are needed

**For bugfix**
- [ ] Unit test is added that covers the bug (if possible)
- [ ] You cursed at the person that introduced the bug

## Reviewer

- [ ] Version number is updated
- [ ] `HISTORY.rst` is updated
- [ ] Function is tested
- [ ] `(docstring)` is added or updated
- [ ] All needed package are in [requirements.txt](requirements.txt)

**For new function**

- [ ] Functions are named based on [Python naming conventions](https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html)
- [ ] Function is added to apidocx.rst
- [ ] New feature is added to `README.md` subsection `Features`
- [ ] New function is imported in all `__init__.py` that are needed

**For bugfix**
- [ ] Unit test is added that covers the bug (if possible)

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -1,11 +1,17 @@
 # History
 
-2.0.0-b1
+2.0.0-b0 (2022-11-22)
```
Comment (wagenrace) [inline HISTORY.rst:3]: 1 header is enough, only the last date will do. This is for referring back to see what changed between releases

Comment (schneiderjan) [reply HISTORY.rst:3]: Just wanted to make clear what's in the PR.

File: tomni/cytosmart_data_format/annotations/annotation/main.py [hunk 2]
```diff
@@ -10,12 +10,14 @@ def __init__(self, id: str, label: str, children: List, parents: List) -> None:
...
-    def to_dict(self) -> dict:
+    def to_dict(self, decimals: int = 2) -> dict:
         """Creates a dictionary in CDF of annotation.
+        Args:
+            decimals (int, optional): The number of decimals to use when rounding. Defaults to 2.
 
         Returns:
-            dict: Dictionary of annotation.
-        """
+            dict: CDF dict.
```
Comment (jellevankerk) [inline tomni/cytosmart_data_format/annotations/annotation/main.py:19]: Should this not 
```suggestion
            dict: CytoSmart Data Format dict.
```

**Review by jellevankerk — CHANGES_REQUESTED** [review]
looks good, added some comments please check them out

File: tomni/cytosmart_data_format/main.py [hunk 3]
```diff
@@ -1,72 +1,150 @@
...
-        return len(self._cdf_data)
+    @classmethod
+    def from_masks(cls, masks: List[np.ndarray]):
+        """could be an option"""
+        pass
 
-    def __eq__(self, __o: object) -> bool:
-        # if {}== {}:
-        """To check equality of to cdf items"""
+    @classmethod
```
[the remark this answers is not in the document]
Comment (schneiderjan) [reply tomni/cytosmart_data_format/main.py:66]: Yes, if we find a good way to compare two CDF objects but not required at the moment.

File: integration_cdf_main.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+#%%
+print(f"__len__: {len(cdf)}")
+
+#%%
+# cdf creates a generator.
+count = 0
+for annotation in cdf:
+    count += 1
+print(f"Count: {count}")
+
```
Comment (jellevankerk) [inline integration_cdf_main.py:24]:
```suggestion
print(f"Dict count:{len(dicts)}")
```
to show that they are equal in length

**Review by jellevankerk — APPROVED** [review]

**Pull request merged**